atlp-rwanda/atlp-pulse-mobile
# Fix: expo app routing

### What does this PR do?

This PR includes fixes to expo routing that was caused by a new package introduced.

### Description of Task to be completed?

Removed the `react-native-dotenv` package that was overwriting expo app router configuration causing the app to not recognize the app folder and not showing any screen.

### How should this be manually tested?

- Run `npm install` to remove the package from your node_modules and run `npm run start` to start the app.
- The app should show the onboarding screen as usual.

### Screenshots

N/A
